omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -389,6 +389,7 @@
 # we'll be using these directories to determine what versions we have installed and
 # whether we need to run pg_upgrade.
 default['private_chef']['postgresql']['enable'] = true
+default['private_chef']['postgresql']['remote'] = false
```

Comment:
That's the path I started down, and I think it's possible but opted for the short path for now. 

We could create a helper method that checks for ipv4/ipv6 link-local addresses as well as iterating the bound IPs on available interfaces.   The  risk here is that it's possible someone will want to manage their own postgres, but still do it on the same box - in which case we'd get a false negative.